docs/readthedocs/usersguide.md

```diff
@@ -186,7 +186,7 @@ The external simulation is loaded via class loader. Thus, SIMONA and the externa
 These steps have to be performed each time updates to the external simulation need to be deployed.
 
 - Execute ``gradle shadowJar`` inside the external simulation project.
-- Copy the resulting *jar* (usually placed inside <external project>/build/libs) to SIMONA/inputData/ext_sim.
+- Copy the resulting *jar* (usually placed inside ``<external project>``/build/libs) to SIMONA/input/ext_sim.
```

Comment:
Please put the whole thing in ``, so

```suggestion
- Copy the resulting *jar* (usually placed inside ``<external project>/build/libs``) to ``./input/ext_sim``.
```

Reply:
I hate to be pendantic here, but please take a look at the whole change I suggested here :)

Reply:
Isn't this suggested change already resolved?